expertise/create_dataset.py

```diff
@@ -36,12 +36,27 @@ def convert_to_list(self, config_invitations):
         assert isinstance(invitations, list), 'Input should be a str or a list'
         return invitations
 
+    def get_paper_notes(self, author_id, dataset_params):
+
+        use_bids_as_expertise = dataset_params.get('bid_as_expertise', False)
+
+        if use_bids_as_expertise:
+            bid_invitation = dataset_params['bid_invitation']
+            paper_invitation = self.config['paper_invitation']
+            bids = openreview.tools.iterget_edges(self.openreview_client, invitation=bid_invitation, tail=author_id)
+            note_ids = [e.head for e in bids if e.label in ['Very High', 'High']]
+            return [n for n in self.openreview_client.get_notes_by_ids(ids=note_ids) if n.invitation == paper_invitation]
```

Comment:
Why can't it be like this?
```py
return [n for n in self.openreview_client.get_notes_by_ids(ids=note_ids)]
```

Reply:
I need to check if the note are not desk rejected or withdrawn.